src/pages/settings/PaymentsPage.js

```diff
@@ -1,24 +1,109 @@
 import React from 'react';
-import HeaderWithCloseButton from '../../components/HeaderWithCloseButton';
-import Navigation from '../../libs/Navigation/Navigation';
+import {Pressable, TextInput, View} from 'react-native';
+import PropTypes from 'prop-types';
+import {withOnyx} from 'react-native-onyx';
+import CONST from '../../CONST';
+import ONYXKEYS from '../../ONYXKEYS';
 import ROUTES from '../../ROUTES';
+import HeaderWithCloseButton from '../../components/HeaderWithCloseButton';
+import Text from '../../components/Text';
 import ScreenWrapper from '../../components/ScreenWrapper';
+import NameValuePair from '../../libs/actions/NameValuePair';
+import {fetch} from '../../libs/actions/User';
+import Navigation from '../../libs/Navigation/Navigation';
+import styles from '../../styles/styles';
+
+const propTypes = {
+    // The PaypalMe Username
+    paypalMeAddress: PropTypes.string,
+};
+
+const defaultProps = {
+    paypalMeAddress: '',
+};
+
+class PaymentsPage extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            paypalMeAddress: props.paypalMeAddress,
+        };
+        this.setPaypalAccount = this.setPaypalAccount.bind(this);
+    }
+
+    componentDidMount() {
+        fetch();
+    }
+
+    componentDidUpdate(prevProps) {
+        if (prevProps.paypalMeAddress !== this.props.paypalMeAddress) {
+            // eslint-disable-next-line react/no-did-update-set-state
```

Comment:
Is there no way to do this outside of `componentDidUpdate` ? If not, can we add a comment like [this one](https://github.com/Expensify/Expensify.cash/blob/257518b71ebd9687e733bcc777e0f07a10fd61c3/src/pages/home/report/ReportTypingIndicator.js#L37)?

Reply:
Yes here `payPalMeUsername` is coming from props, and when the pusher event is set up for `expensify_PayPalMeAddress` It will be updated in realtime if the user is on the same screen on multiple platforms.

I have added the comment `// Suppressing because this is within a conditional, and hence we won't run into an infinite loop`